sources/pmwebapi_test.go

```diff
@@ -92,12 +161,12 @@ func TestLabelTypes(t *testing.T) {
 		t.Fatalf("Failed when type is any other than counter or empty. Expected: %s, Got: %s", expected, response)
 	}
 
-	units = "87"
-	pcpType = "countER"
-	name = "k.a.l"
-	response = typeLabel(units, pcpType, name)
-	expected = "k_a_l_87_total"
-	if response != expected {
-		t.Fatalf("Failed when there were periods in the name. Expected: %s, Got: %s", expected, response)
-	}
+	//units = "87"
```

Comment:
Probably wanna delete these if you don't want them anymore rather than commenting them out.